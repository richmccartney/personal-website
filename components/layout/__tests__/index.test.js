import React from 'react'
import renderer from 'react-test-renderer'
import { mount } from 'enzyme'

import Layout from '../index'

describe('Layout:', () => {
  const wrapper = mount(<Layout>Hello</Layout>)

  test('renders snapshot as expected', () => {
    const component = renderer.create(<Layout />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders as <div> as expected', () => {
    const container = wrapper.find('div')
    expect(container.is('div')).toBe(true)
  })

  it('renders with <header> as expected', () => {
    const header = wrapper.find('header')
    expect(header.is('header')).toBe(true)
  })
})
