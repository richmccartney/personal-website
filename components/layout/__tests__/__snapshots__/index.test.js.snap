// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout: renders snapshot as expected 1`] = `
<div
  className="css-cglpfw"
>
  <header
    className="css-dhve4b"
  >
    <a
      className="css-kl9a67"
      href="/"
    >
      Richard McCartney
    </a>
  </header>
</div>
`;
